license/src/com/untangle/app/license/LicenseManagerImpl.java

```diff
@@ -1010,12 +1011,33 @@ private void _syncLicensesWithServer()
                     _saveSettings(this.settings);
                 }
             }
+            _cleanLicenseFiles();
             _runAppManagerSync();
         }
 
         logger.info("Reloading licenses... done" );
     }
 
+    /**
+     * Pruning old license files, keeping last 5 license files.
+     */
+    private void _cleanLicenseFiles() {
+        File licenseDirectory = new File(System.getProperty("uvm.conf.dir") + "/licenses/");
+        try {
+            if (licenseDirectory.isDirectory()) {
+                File[] files = licenseDirectory.listFiles();
+                if (files != null && files.length > 5) {
+                    Arrays.stream(files)
```

Comment:
Looks really good. Can we have a info log here that it is cleaning the old license files ?